### PR TITLE
Validate gender selections, add log out feature.

### PR DESCRIPTION
**Pull request by starkshaw**

### CODE REVIEW

**Comment by starkshaw** [comment]
Bugs detected.

**Comment by starkshaw** [comment]
Bug fixed.

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -36,8 +36,16 @@
             %li
               %a.active{:href => "/"} Home
           %ul.nav.navbar-nav.navbar-right.musikore-navbar-item
-            %li
-              %a{:href => "/login"} Log in
+            - if user_signed_in?
+              %li
+                = current_user.username
+              %li
+                = link_to "Edit Profile", "/edit"
```
Comment (danielpaul) [inline app/views/layouts/application.html.haml]: Use named route paths instead of hard coding paths.

Comment (starkshaw) [inline app/views/layouts/application.html.haml]: How?

File: app/views/layouts/application.html.haml [hunk 2]
```diff
@@ -36,8 +36,16 @@
...
+              %li
+                = link_to "Log out", destroy_user_session_path, method: :delete
```
Comment (danielpaul) [inline app/views/layouts/application.html.haml:45]: @starkshaw similar to this. To find all routes run `rake routes` and get the name from there and add `_path` in the end of that named route.

**Comment by starkshaw** [comment]
Fixed. @danielpaul

**Comment by danielpaul** [comment]
:+1: Looks good. Good for a merge @starkshaw

**Pull request merged**